deploy/helm/kubecf/assets/operations/job_moving/move_log_cache.yaml

```diff
@@ -0,0 +1,22 @@
+- type: remove
+  path: /instance_groups/name=doppler/jobs/name=log-cache
+- type: remove
+  path: /instance_groups/name=doppler/jobs/name=log-cache-gateway
+- type: remove
+  path: /instance_groups/name=doppler/jobs/name=log-cache-nozzle
+- type: remove
+  path: /instance_groups/name=doppler/jobs/name=log-cache-cf-auth-proxy
+- type: remove
+  path: /instance_groups/name=doppler/jobs/name=route_registrar
+- type: replace
+  path: /instance_groups/name=log-api:after
+  value:
+    name: log-cache
+    instances: 1
+    stemcell: default
+    update:
+      serial: false
+    jobs:
+    {{- range $path, $bytes := $.Files.Glob "assets/jobs/log_cache_*" }}
+    - {{- $.Files.Get $path | nindent 6 }}
+    {{- end }}
```

Comment:
We never used `$bytes` because it returns `[]uint8`, not string. We can make it work with `toString`, though. I'll prepare a separate PR for fixing all the instances where we could do it, which goes way beyond this current PR.